generate/generate.go

```diff
@@ -461,14 +456,9 @@ func (g *Generator) ClearLinuxSysctl() {
 }
 
 // AddLinuxSysctl adds a new sysctl config into g.spec.Linux.Sysctl.
-func (g *Generator) AddLinuxSysctl(s string) error {
+func (g *Generator) AddLinuxSysctl(key, value string) error {
```

Comment:
It seems this function does not need `return error` any more. Same as other AddXXXX() function.